Dockerfiles/backend.Dockerfile

```diff
@@ -3,7 +3,7 @@
 #############################
 
 # Set official image -- parent image
-FROM python:3.11-alpine as base
+FROM python:3.12-alpine as base
```

Comment:
Do we want to update some corresponding Python stuff in dds_cli? 

https://github.com/ScilifelabDataCentre/dds_cli/blob/7fefe4c083883f544f1b5a32d34b1ea98de89ff2/setup.py#L27 

https://github.com/ScilifelabDataCentre/dds_cli/blob/7fefe4c083883f544f1b5a32d34b1ea98de89ff2/WINDOWS.md?plain=1#L3

Reply:
That would need to go in a different PR anyway. Maybe better to comment it on the jira task?

Reply:
Yup, more just a thought that this should be part of this ticket (but not this PR).